### DISCUSSION TITLE
Installation with conda installs old spacy version

### DISCUSSION THREAD

**Discussion by hp52** [explosion/spaCy#7856] [Help: Installation] [answered]
Hi,

I'm trying to install spacy with conda:
If I use this command

`conda install -c conda-forge spacy`

it installs spacy 2.3.4 instead of the latest version.

When I try 

`conda install -c conda-forge spacy=3.0.5`

it keeps saying
```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
```
However,
`conda search spacy --channel conda-forge`
indicates that spacy 3.0.5 should be available.

What am I doing wrong?
Thanks for your help.

**Comment by hp52** [comment, marked as answer]
Nevermind, I got it solved. It was a conda problem. After changing from the base environment to another environment, it worked fine.

**Reply by codingbutstillalive** [reply]
Do you have an explanation of why this happened with the base environment but not with another one?

I have the same problem, but I used a clone of the base environment - which I assigned another name. So technically it wasn't the base environment. But it only installs the old version of spaCy into it.

**Reply by polm** [reply]
I think that the cause might be that the base environment has old packages in it, and it looks for a version of spaCy it can use without upgrading those. Using a blank environment has nothing installed so that doesn't happen. Not really sure though.
